database/package.json

```diff
@@ -6,10 +6,6 @@
   "license": "MIT",
   "scripts": {
     "serve": "docker stop localdev-db; docker rm localdev-db; docker run --detach --rm -p 8888:5984 --name localdev-db five-fifths-voter/db",
-    "build": "docker build -t five-fifths-voter/db .",
-    "postinstall": "cd .. && husky install"
-  },
-  "devDependencies": {
-    "husky": "^7.0.4"
```

Comment:
😄 So we just recently added this as a placeholder to enable some real work in precommits.
#285 
Can you revert this change. I still need to open a new issue asking for at least a lint check in precommit.

Reply:
Ok, husky restored removing removal commits, I will inspect other changes tomorrow